content/about.md

```diff
@@ -24,4 +24,13 @@ In 2015, researchers discovered that ~20% of the Alexa top million domains which
 
 These results are the motivation for this project.
 
-<small><sup>[1]</sup> Neither Snow Nor Rain Nor MITM . . . An Empirical Analysis of Email Delivery Security https://zakird.com/papers/mail.pdf</small>
+<strong class="js-yes">Percent of scans on starttls-everywhere.org supporting MTA-STS</strong>
+<canvas id="visitors-chart" height="100"></canvas>
+
+<strong class="js-yes">Domains in the top million<sup>[2]</sup> supporting MTA-STS</strong>
+<canvas id="top-million-chart" height="100"></canvas>
+
```

Comment:
Let's move these charts up above the `Statistics` section and add the following text before the charts:

```html
<h5>MTA-STS adoption and integration</h5>
MTA-STS is a new Internet standard for e-mail that has many of the same goals as STARTTLS Everywhere. We're tracking its adoption across the top million domains and users of our site. Here are some growth statistics about the deployment of MTA-STS!
```

I also realized I never pushed the new drafts of MTA-STS FAQ, etc. Let me see if I can find the branch!